src/NuGetForUnity/Editor/NugetPackageAssetPostprocessor.cs

```diff
@@ -67,6 +67,20 @@ private void OnPreprocessAsset()
             LogResults(results);
         }
 
+        private static void OnPostprocessAllAssets(string[] importedAssets,
+            string[] deletedAssets,
+            string[] movedAssets,
+            string[] movedFromAssetPaths)
+        {
+            if (Array.IndexOf(importedAssets, NugetHelper.PackagesConfigFilePath) < 0)
```

Comment:
Fixed this by expanding both paths to full before checking. The `PackagesConfigFilePath` will be moved to NuGetConfig in another PR after this is merged.
Maybe I am missing something but I am not sure how to Unit Test this method...but you were right, it was not working properly initially.

Reply:
I prefer using absolute path as it can't be misused e.g. wenn you parse it to a different tool. Also it generates better exceptions e.g. when you get a DirectoryNotFoundException it is much easier to understand what is happening when the path is absolut.